components/admin_console/billing/billing_subscriptions.tsx

```diff
@@ -51,12 +52,27 @@ const BillingSubscriptions: React.FC<Props> = () => {
     const currentUser = useSelector((state: GlobalState) => getCurrentUser(state));
     const isCloud = useSelector((state: GlobalState) => getLicense(state).Cloud === 'true');
     const subscription = useSelector((state: GlobalState) => state.entities.cloud.subscription);
+    const customer = useSelector((state: GlobalState) => state.entities.cloud.customer);
     const products = useSelector((state: GlobalState) => state.entities.cloud.products);
     const getCategory = makeGetCategory();
     const preferences = useSelector<GlobalState, PreferenceType[]>((state) => getCategory(state, Preferences.ADMIN_CLOUD_UPGRADE_PANEL));
 
     const contactSalesLink = useSelector((state: GlobalState) => getCloudContactUsLink(state, InquiryType.Sales));
 
+    const isCreditCardExpired = () => {
+        if (!customer) {
+            return false;
+        }
+
+        // Will developers ever learn? :D
+        const expiryYear = customer.payment_method.exp_year + 2000;
```

Comment:
Hehehe, in defense of devs, last two digits of CC is probably standard

Reply:
You'd think we'd have learned after the whole Y2K fiasco :P